src/Schema/Attribute/Attribute.php

```diff
@@ -6,69 +6,124 @@
 
 class Attribute extends Item implements AttributeSingle
 {
+    /**
+     * @var static|null
+     */
     protected $fixed;
 
+    /**
+     * @var static|null
```

Comment:
here is `string` instead of `static|null`

Reply:
as above.